ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -283,6 +309,34 @@ impl From<Invitation> for InvitationNext {
     }
 }
 
+#[derive(Debug, GraphQLObject)]
+#[graphql(context = Context)]
+pub struct JobRun {
+    pub id: juniper::ID,
+    pub job_name: String,
+    pub start_time: DateTime<Utc>,
+    pub finish_time: Option<DateTime<Utc>>,
+    pub exit_code: Option<i32>,
+    pub stdout: String,
+    pub stderr: String,
```

Comment:
Currently, return result does not contain `updated_at` & `created_at`, not sure if we should add them, considering there already exist `start_tim` & `finish_time`.